qhub/initialize.py

```diff
@@ -97,6 +98,26 @@
     },
 }
 
+AZURE = {
+    "project": "PLACEHOLDER",
+    "region": "Central US",
```

Comment:
Just noting that the vast majority of public data from Azure's public datasets program will be hosted in the West Europe. If qhub's users are interested in that kind of data, and I suspect many are, the best default might be "West Europe".

The current catalog is at https://github.com/microsoft/AIforEarthDataSets.